default/templates/layouts/book.tpl

```diff
@@ -71,9 +74,10 @@
             <apply template="components/metadata" />
             <apply template="/templates/hooks/note-end" />
           </main>
+          <apply template="/templates/hooks/after-main" />
```

Comment:
I see some `<apply template>` have `/` prefix, while some dont. Is there a major difference?

Reply:
IIUC - one is relative, and the other is absolute (from templates/ directory).